### ISSUE TITLE
Feature Idea - CBERS From AWS

### ISSUE THREAD

**Issue by fishish1** [16EAGLE/getSpatialData#69]
CBERS Data is similar to Landsat data. With a second CBERS-4(a) satellite currently undergoing validation, data from CBERS could helpfully supplement Landsat and Sentinel data. 

CBERS data is available in an AWS bucket, cbers-pds. Accessing this data would require interacting with the Amazon CLI (python) and an AWS account with the correct privileges. I could provide steps to help access the s3 bucket.

Challenges include: CBERS doesn't appear to have cloud masks included

**Comment by 16EAGLE** [comment]
Thanks for the suggestion. Is the CBERS AWS bucket freely available or is it run on a requester-pays basis?

**Comment by fishish1** [comment]
Yes. I'm not exactly sure how affordable that makes it for the average human.
